pkg/scheduler/apis/config/types.go

```diff
@@ -24,6 +24,8 @@ import (
 const (
 	// SameRegion Strategy determines a seed candidate for a shoot only if the cloud profile and region are identical
 	SameRegion CandidateDeterminationStrategy = "SameRegion"
+	// BestRegion Strategy prefers a matching region, only if no such seed exsists it tries to find the nearest one.
+	BestRegion CandidateDeterminationStrategy = "BestRegion"
```

Comment:
Can you also add it here: https://github.com/gardener/gardener/blob/99bf01f0935d1333d284c60251717ea62102f7e4/pkg/scheduler/apis/config/v1alpha1/types.go#L26